### PR TITLE
Bug/fix styling mistake in child dashboard

### PR DESCRIPTION
**Pull request by OniWarrior**
Fixed a big styling mistake with child dashboard.

### CODE REVIEW

File: src/components/pages/Gamemode/Gamemode.js [hunk 1]
```diff
@@ -98,15 +98,11 @@ const Gamemode = ({ ...props }) => {
           !props.child.gamemode.sp && (
             <div className="dash-container">
               <Header />
-              <div className="adventure-passport-container">
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:101]: nice looks much cleaner

File: src/styles/less/ChildDashboard.less [hunk 2]
```diff
@@ -25,7 +25,7 @@
   .adventure-passport,
   .trophy-room,
   .leaderboard {
-    border: 2px solid @header-nero;
+    border: 5px solid @header-nero;
```
Comment (just-kato) [inline src/styles/less/ChildDashboard.less:28]: Love a solid border. Good job

**Review by mollybee — APPROVED** [review]
Good speedy fix on the styling. Looks good to me.

File: src/styles/less/ChildDashboard.less [hunk 3]
```diff
@@ -135,10 +135,18 @@
     }
   }
 
-  .adventure-passport-container{
+  .single-button-container{
```
Comment (just-kato) [inline src/styles/less/ChildDashboard.less:138]: nice rearrangement

File: src/styles/less/ChildDashboard.less [hunk 4]
```diff
@@ -210,15 +210,20 @@
 }
 
 .adventure-passport:hover {
-  animation:scale-up-singleplayer-btn linear 0.3s;
-  transform: scale(1.01);
+  opacity: 100%;
```
Comment (just-kato) [inline src/styles/less/ChildDashboard.less:214]: good work!

File: src/styles/less/ChildDashboard.less [hunk 5]
```diff
@@ -156,6 +156,7 @@
     margin-left:2.5%;
     width:95%;
     min-height: 50vh;
+    border: 5px solid @header-nero;
```
Comment (just-kato) [inline src/styles/less/ChildDashboard.less:159]: Border looks great

**Review by just-kato — APPROVED** [review]
as always great job Stephen

**Pull request merged**